tsdb/importer/importer.go

```diff
@@ -546,3 +564,10 @@ func getBlockIndex(current int64, times []blockTimestampPair) int {
 	}
 	return -1
 }
+func reverse(numbers []int64) []int64 {
```

Comment:
you can do this with the std lib - SortInts and Revers

Reply:
I did not use those function in case the durations passed in were unsorted!